department-of-veterans-affairs/vets-website
# feat: filter quick submit group out

## Summary

- Filters out group5 if they are present in the Communication Preferences.
- This group of notifications called 'Quick Submit' is not ready for production yet, and so this is going to be blocked until then.

## Related issue(s)

- https://github.com/department-of-veterans-affairs/va.gov-team/issues/55658
- Slack thread with the group number being confirmed

## Testing done

Added e2e tests for the group being hidden and shown via feature toggle

_To test locally_
 ``` bash
yarn watch --env entry=profile
```
then
``` bash
yarn mock-api --responses src/applications/personalization/profile/mocks/server.js
```
then visit `http://localhost:3001/profile/notifications` to view page and verify that the group for QuickSubmit does not appear.

You may then turn the feature toggle on via the `src/applications/personalization/profile/mocks/server.js` file so that the `profileShowQuickSubmitNotificationSetting` toggle is set to `true`. Once you do that and refresh the page you will see the quick submit notification group appear as expected.

## Screenshots

|         | False | True |
| ------- | ------ | ----- |
|  | ![Screenshot 2023-05-10 at 8 28 00 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/8332986/5e7521d7-8c0d-48ac-ab44-a93218d3a46a) | ![Screenshot 2023-05-10 at 8 27 42 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/8332986/9f984ea5-3f70-42b6-a830-a611682d99e1) |

## What areas of the site does it impact?

Profile - Notification Settings

## Acceptance criteria

- [x] Group hidden by feature toggle

### Quality Assurance & Testing


### Error Handling
